task.sql

```diff
@@ -1,33 +1,55 @@
--- Create database and tables
-
 CREATE DATABASE ShopDB;
```

Comment:
Consider adding `DROP DATABASE IF EXISTS ShopDB;` before creating the database to ensure it is dropped if it already exists.